notebooks/Analyze_Model_Outputs.ipynb

```diff
@@ -844,83 +844,736 @@
      "data": {
```

Comment:
<p>Is this supposed to be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>That's strange, it looks ok on my machine and on Github's own renderer. I guess ReviewNB is mangling HTML output in a way that we didn't expect. From a quick poke at the DOM tree for this page, it looks like ReviewNB is trying to run the JavaScript in the visualization and ending up with an empty div somehow.</p><p> </p><p>FYI @<a href="https://github.com/PokkeFe" target="_blank">PokkeFe</a></p>